src/MixpanelEventForwarder.js

```diff
@@ -13,6 +13,15 @@
 //  See the License for the specific language governing permissions and
 //  limitations under the License.
 
+// ** Glossary of terms
+// Mixpanel Terms:
+// user_id: A unique identifier used by the Mixpanel.identify method to identify a unique user
```

Comment:
```suggestion
// user_id: A unique identifier used by the Mixpanel.identify method to identify a unique user.  This will map to what an mParticle customer selects in the UI, which translates to the forwarderSettings.userIdentificationType.
```